docs/resources/Application.md

```diff
@@ -19,11 +19,13 @@
 | verify_key                         | string                                                                     | the hex encoded key for verification in interactions and the GameSDK's [GetTicket](#DOCS_GAME_SDK_APPLICATIONS/getticket)                                                   |
 | team                               | ?[team](#DOCS_TOPICS_TEAMS/data-models-team-object) object                 | if the application belongs to a team, this will be a list of the members of that team                                                                                       |
-| guild_id?                          | snowflake                                                                  | if this application is a game sold on Discord, this field will be the guild to which it has been linked                                                                     |
+| guild_id?                          | snowflake                                                                  | guild associated with the bot. For example a developer support server.                                                                                                      |
+| guild?                             | partial [guild](#DOCS_RESOURCES_GUILD/guild-object) object                 | a partial object of the associated guild                                                                                                                                    |
 | primary_sku_id?                    | snowflake                                                                  | if this application is a game sold on Discord, this field will be the id of the "Game SKU" that is created, if exists                                                       |
 | slug?                              | string                                                                     | if this application is a game sold on Discord, this field will be the URL slug that links to the store page                                                                 |
 | cover_image?                       | string                                                                     | the application's default rich presence invite [cover image hash](#DOCS_REFERENCE/image-formatting)                                                                         |
 | flags?                             | integer                                                                    | the application's public [flags](#DOCS_RESOURCES_APPLICATION/application-object-application-flags)                                                                          |
+| approximate_guild_count?           | integer                                                                    | an approximate count of the bots guild membership.                                                                                                                          |
```

Comment:
```suggestion
| approximate_guild_count?           | integer                                                                    | an approximate count of the guilds an application is a member of                                                                                                                          |
```

Reply:
"an application" doesn't really match the rest, it should be "the application", like how the current docs say "the bots"

Reply:
maybe something like `an approximate count of the guilds the bot was added to`